packages/frontend-test-utils/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@backstage/frontend-test-utils",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "web-library"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "devDependencies": {
+    "@backstage/cli": "workspace:^",
+    "@testing-library/jest-dom": "^6.0.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@backstage/frontend-app-api": "workspace:^",
```

Comment:
Just double-checking, it is fine to have a dependency on the app package here,right?

Reply:
Good check 😁 👍, but yep, it's on the users of this package to make sure it's a dev dep though